h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -166,19 +169,25 @@ export default function AnnotationCard({
               </div>
             </header>
 
-            <StyledText>
-              <blockquote className="hover:border-l-blue-quote">
-                {quote(annotation)}
-              </blockquote>
-            </StyledText>
+            {annoQuote && (
+              <Excerpt collapsedHeight={40} overflowThreshold={20}>
+                <StyledText>
+                  <blockquote className="hover:border-l-blue-quote">
+                    {annoQuote}
+                  </blockquote>
+                </StyledText>
+              </Excerpt>
+            )}
 
-            <MarkdownView
-              markdown={annotation.text}
-              mentions={annotation.mentions}
-              mentionMode="username"
-              mentionsEnabled
-              classes="text-color-text"
-            />
+            <Excerpt collapsedHeight={194} overflowThreshold={20}>
```

Comment:
In my testing I found this cut off the text in the middle of the line. This is always a possibility as long as the cut-off point is chosen based on a pixel size, since the content could include images etc. of arbitrary height. However we probably should choose a multiple of the line height (ie. a value in "em"-units) by default, so this doesn't happen if the content is just plain text.

<img width="733" height="428" alt="Image" src="https://github.com/user-attachments/assets/5c269f56-fa0e-4a8c-8303-a2162203be01" />

Reply:
This would be probably mitigated by displaying a shadow on the lower side of the collapsed content.